srcpkgs/nvimpager/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'nvimpager'
+pkgname=nvimpager
+version=0.6
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+make_build_args="PREFIX=/usr"
```

Comment:
The preix should already be part of the makefile arguments in the `gnu-makefile` build style, so I don't think it needs to be included.